### PR TITLE
PGD 5.4.1 Release Notes

### PR DESCRIPTION
**Pull request by djw-m**
## What Changed?

Dates for 5.4.1 release

### CODE REVIEW

**Review by irionr — APPROVED** [review]
LGTM, but we don't have a confirmation yet from Natalia.

**Review by irionr — APPROVED** [review]

File: product_docs/docs/pgd/5/rel_notes/pgd_5.4.1_rel_notes.mdx [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+|-----------|---------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------|
+| BDR       | 5.4.1  | Fixed WAL retention logic, when a node is parted which impacted PGD 5.4 and PG 16 nodes. For example, if you were to delete a VM as part of a fault injection exercise, the replication slot will reconnect in a timely manner                                                                                                                                                                                                                                                   |                  |
```
Comment (mwanner) [inline product_docs/docs/pgd/5/rel_notes/pgd_5.4.1_rel_notes.mdx:19]: This fix is not related to parting nodes.

Not sure whether or not it matches the draft Email announcement that Natalia prepared here:
https://docs.google.com/document/d/1eQ2I45jfuEF5tlVxhEsazEZKmbTdWmOUuftIfxDteMc

Let's coordinate and figure the exact wording.

Comment (mwanner) [reply product_docs/docs/pgd/5/rel_notes/pgd_5.4.1_rel_notes.mdx:19]: Please see the proposed/planned announcement email here:
https://docs.google.com/document/d/1eQ2I45jfuEF5tlVxhEsazEZKmbTdWmOUuftIfxDteMc

Date will need to be adjusted to April 3rd.

Full text I originally proposed (2nd para can be additional explanation in the release notes)
```
Fixed WAL retention logic: prevent a PGD node from running out of disk space
due to a bug in 5.4.0 in combination with Postgres 16, PGE 16, and EPAS 16.

Make sure WAL gets properly cleaned even after 4 GB of WAL produced on a
node. A change in 5.4.0 caused WAL to be retained forever after that point.
This issue affects only release PGD 5.4.0  in combination with Postgres 16,
PGE 16, and EPAS 16. 
```
